### PR TITLE
test(popover-menu): Adding unit tests

### PR DESCRIPTION
**Pull request by alfredskeden**
**Describe pull-request**  
Adds unit tests for popover menu.

**Solving issue**  
Fixes: [CDEP-3008](https://tegel.atlassian.net/browse/CDEP-3008)

**How to test**
1. Checkout code
2. Check that `npm run test` works and completes all tests

[CDEP-3008]: https://tegel.atlassian.net/browse/CDEP-3008?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/core/src/components/popover-menu/test/icons-fluid/popover-menu.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    /* Check diff on screenshot */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('menu item "The menu with adjusts to the widest word" exists', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const tdsMenuItemListLongText = page.getByText('The menu width adjusts to the widest word');
+    await expect(tdsMenuItemListLongText).toHaveCount(1);
+    await expect(tdsMenuItemListLongText).toBeHidden();
```
Comment (theJohnnyMe) [inline packages/core/src/components/popover-menu/test/icons-fluid/popover-menu.e2e.ts:20]: Why is "toBeHidden" here?

Comment (alfredskeden) [reply packages/core/src/components/popover-menu/test/icons-fluid/popover-menu.e2e.ts:20]: Without clicking the button to open the menu it is hidden so I also make sure that its hidden when I haven't opened it. I guess it does not reflect the test exactly tho. It says "exists" so

Comment (theJohnnyMe) [reply packages/core/src/components/popover-menu/test/icons-fluid/popover-menu.e2e.ts:20]: OK, I get you - you are checking element in the DOM, but you do not really care if it up in front of the eyes or not, which also is true if user did not clicked on button. Makes sense, all good.

**Review by theJohnnyMe — APPROVED** [review]

**Review by timrombergjakobsson — APPROVED** [review]
:shipit:

**Pull request merged**